docs/src/views/FAQ.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react'
+import DocPage from '../components/DocPage/DocPage'
+import GuidesNavigationFooter from '../components/GuidesNavigationFooter'
+import CodeSnippet from '../components/CodeSnippet'
+
+import { Header } from '@stardust-ui/react'
+
+export default () => (
+  <DocPage title="FAQ">
+    <Header as="h2" content="General" />
+    <Header as="h3" content="Does Stardust support mobile?" />
+    <p>No. Scope is limited to Web / Desktop at present.</p>
+
+    <Header as="h3" content="How can @mixins be used in the Stardust's styles" />
+    <p>
+      As the styles in Stardust are defined using CSS in JS, mixins can be defined as simple
+      functions that can be reused on multiple places.
+    </p>
+
+    <Header as="h3" content="Links and React-Router" />
```

Comment:
I think that this should be extracted to a separate page, you can use SUIR docs: https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/src/pages/Augmentation.mdx

Reply:
You mean as a page describing the 'as' property?  If so, I agree we should have that. But I had this specific question - so we could duplicate this information in the FAQ.

Reply:
> You mean as a page describing the 'as' property? 

Exactly